frontend/src/metabase/admin/settings/components/widgets/PublicLinksListing/PublicLinksListing.jsx

```diff
@@ -142,6 +151,26 @@ export const PublicLinksQuestionListing = () => (
   />
 );
 
+const mapStateToProps = state => ({
+  siteUrl: getSetting(state, "site-url"),
+});
+
+export const PublicLinksActionListing = connect(mapStateToProps)(
+  function PublicLinksActionListing({ siteUrl }) {
```

Comment:
@ranquild just to make the structure similar to other `PublicLinksXxxxListing` I opt to use the inner function name similar to other places like:
https://github.com/metabase/metabase/blob/7b6979276f25ffbff9c0f05d1ba96da5ca622647/enterprise/frontend/src/metabase-enterprise/whitelabel/components/BrandColorSettings/BrandColorSettings.tsx#L96

And it shows `displayName` correctly now.
<img width="873" alt="image" src="https://user-images.githubusercontent.com/1937582/218190488-0a2a2260-881f-49d8-b4d0-dbfa8cc514a2.png">